aten/src/ATen/native/native_functions.yaml

```diff
@@ -6943,6 +6939,7 @@
   dispatch:
     ZeroTensor, CPU, CUDA, Meta, QuantizedCPU, QuantizedCUDA, MPS: view
     MkldnnCPU: mkldnn_view
+    NestedTensorCPU, NestedTensorCUDA: view_nested
```

Comment:
> hmm I'm not sure that this is what we want.
> 
> I think of `CompositeImplicitAutograd` as "all backends + layout keys, plus all autograd keys".
> 
> And then now, I think of `CompositeImplicitAutogradNestedTensor` as "all nested backends, plus the nested autograd key".
> 
> So in that view, `CompositeImplicitAutograd` should be a strict superset of `CompositeImplicitAutogradNestedTensor`: if a kernel is registered directly to `CompositeImplicitAutograd`, that should still cause all of the nested tensor dispatch keys to get that kernel too.
> 
> We also have a handful of native_function entries that explicit use `CompositeImplicitAutograd`, so this would cause NestedTensor to lose those kernels. e.g.
> 
> https://github.com/pytorch/pytorch/blob/1f38225b56b873c944196241ea448445a61798fd/aten/src/ATen/native/native_functions.yaml#L3951
> 
> .